client/js/nonprofits/donate/wizard.js

```diff
@@ -97,19 +99,22 @@ const init = params$ => {
   const dedicationParams$ = flyd.zip([state.infoStep.savedDedicatee$, state.infoStep.savedSupp$, state.paymentStep.paid$])
   const savedDedication$ = flyd.flatMap(R.apply(postDedication), dedicationParams$)
 
+  // handle when the wizard is marked completed, i.e. the finish button hasn't been clicked but charge is completed
+  flyd.lift(
+    (_completedEv, params) => {
+      if (params['skipFinish']) {
```

Comment:
Confirming this is exactly what we want - just presence of the parameter (and not that it’s actually “t” or truthy)?

Reply:
It is, we don't care what the value is. If it's there, we skip the finish screen.